### PR TITLE
Adding my student file and associated media

### PR DESCRIPTION
**Pull request by RyanKim124**

### CODE REVIEW

**Review by BardiaAnvari — DISMISSED** [review]
LGTM!

**Review by anirudhb — CHANGES_REQUESTED** [review]
Please add your Classroom.java changes

**Comment by RyanKim124** [comment]
Added the classroom file changes, should be good now imo

**Review by anirudhb — APPROVED** [review]
lgtm

File: Classroom.java [hunk 1]
```diff
@@ -152,6 +152,10 @@ private void prepare()
         KilgoreTrout kilgoretrout = new KilgoreTrout();
         addObject(kilgoretrout,2,3);
         kilgoretrout.assignSeat();
+        
+        RyanKim ryankim = new RyanKim();
+        addObject(ryankim,6,4);
+        kilgoretrout.assignSeat();
```
Comment (anirudhb) [inline Classroom.java]:
```suggestion
        ryankim.assignSeat();
```

**Review by anirudhb — CHANGES_REQUESTED** [review]

**Review by anirudhb — APPROVED** [review]
lgtm

**Pull request merged**